apps/apae/src/components/sidebar/sidebar.tsx

```diff
@@ -173,11 +174,26 @@ export function AppSidebar() {
                           : "text-[#0D4F97] hover:bg-[#0D4F97] hover:!text-white"
                       )}`}
                     >
-                      <TasklistIcon size={16} />
+                      <PulseIcon size={16} />
```

Comment:
```suggestion
                      <SquareActivity size={16} />
```

Preferível o uso do ícone originalmente escolhido para esta tela, na branch dev-patient.

Reply:
Corrigido.